pkg/serving/revision_template_test.go

```diff
@@ -151,3 +151,89 @@ func TestPort(t *testing.T) {
 	revisionSpec.PodSpec.Containers[0].Ports = []corev1.ContainerPort{port}
 	assert.Equal(t, (int32)(42), *Port(revisionSpec))
 }
+
+func TestContainerStatus(t *testing.T) {
+	tests := []struct {
+		name string
+		rev  *servingv1.Revision
+		want *servingv1.ContainerStatus
+	}{
+		{
+			"no container",
+			&servingv1.Revision{
+				Spec: servingv1.RevisionSpec{},
+			},
+			nil,
+		},
+		{
+			"no container",
+			&servingv1.Revision{
+				Spec: servingv1.RevisionSpec{},
+			},
+			nil,
+		},
```

Comment:
Is it a duplicate? Maybe that should be  `No ImageDigest`?

Reply:
Anyway lets add not empty status without image digest. That was my initial thought. :)

Reply:
oops, yes. Removed the duplicate test case and modified the `no container` case with a non-empty `Status` field.